tests/unit/amo/reducers/test_addonsByAuthors.js

```diff
@@ -812,30 +794,30 @@ describe(__filename, () => {
         undefined,
         loadAddonsByAuthors({
           addons: Object.values(addons),
-          authorUsernames: ['fakeUsername'],
+          authorIds: [randomAuthorId1],
           count: Object.values(addons).length,
           pageSize: EXTENSIONS_BY_AUTHORS_PAGE_SIZE,
         }),
       );
 
-      expect(getAddonsForUsernames(state, ['nobody'])).toBeNull();
+      expect(getAddonsForAuthorIds(state, [999])).toBeNull();
```

Comment:
How about `randomAuthorId1 + 1` instead of `999`. Then we guarantee they're different.